### PR TITLE
fix(setup): Include MailHog setup instructions

### LINKED ISSUE #15838: Local login with passwordless
Passwordless login has been implemented! 🎉 Great work @raisedadead 👍👍 

Now that we have passwordless login, logging in to your local development instance has changed quite a bit. @ogzcoder and @zk433 have already run into this. We'll need to update our documentation on signing up and logging in locally.

Relevant docs:
[CONTRIBUTING.md](https://github.com/freeCodeCamp/freeCodeCamp/blob/abbdb28228331e418ce383f0443b0c9f021986a6/CONTRIBUTING.md)
[Forum guide: How to Log in To your local instance of Free Code Camp](https://forum.freecodecamp.org/t/how-to-log-in-to-your-local-instance-of-free-code-camp/19552)

### PR DESCRIPTION
**Pull request by Spidy88**
#### Pre-Submission Checklist
- [x] Your pull request targets the `staging` branch of freeCodeCamp.
- [x] Branch starts with either `fix/`, `feature/`, or `translate/` (e.g. `fix/signin-issue`)
- [x] You have only one commit (if not, [squash](http://forum.freecodecamp.org/t/how-to-squash-multiple-commits-into-one-with-git/13231) them into one commit).
- [x] All new and existing tests pass the command `npm test`. Use `git commit --amend` to amend any fixes.

#### Type of Change
- [x] Small bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds new functionality)
- [ ] Breaking change (fix or feature that would change existing functionality)
- [ ] Add new translation (feature adding new translations)

#### Checklist:
- [x] Tested changes locally.
- [x] Closes currently open issue: Closes #15838

#### Description
With the changes to passwordless account creation, contributors are
unable to create an account locally. MailHog spins up a local SMPT
server that can catch all outgoing emails, allowing contributors to
successfully create accounts and see any other emails generated locally.

### CODE REVIEW

**Comment by raisedadead** [comment]
Hi @Spidy88 thanks a lot for the PR. 

Can you possibly update the `docker-compose` file? I believe this is a would be a good addition.

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -189,6 +190,12 @@ Note: Not all keys are required, to run the app locally, however `MONGOHQ_URL` i
 
 You can leave the other keys as they are. Keep in mind if you want to use more services you'll have to get your own API keys for those services and edit those entries accordingly in the .env file.
 
+Next you should setup MailHog, a local SMTP mail server that will catch all the outgoing freeCodeCamp messages generated locally. How you start up MailHog is dependent upon your OS, but here's an example for MacOS with Brew.
+
+```bash
+brew services start mailhog
+```
+
```
Comment (raisedadead) [inline CONTRIBUTING.md]: Please add additional info about opening up the localhost web view of the service available at 8025 port, or something on similar lines.

**Review by raisedadead — CHANGES_REQUESTED** [review]

**Comment by Spidy88** [comment]
@raisedadead Ya I can add the mailhog container to the docker-compose. I was avoiding it because the Docker setup seemed broken when I tried it and I just wanted a quick fix for contributors. Let me see if I can fix the Docker setup.

**Comment by raisedadead** [comment]
Ah.. I see in that case lets make a fresh PR for the docker setup when we are ready, for now, could you just add the instruction about web view and we should be good to merge.

Thanks a lot.

**Review by raisedadead — APPROVED** [review]

**Comment by raisedadead** [comment]
#### :sparkles: :sparkles: **LGTM.** :sparkles: :sparkles: 

Thanks @Spidy88 for this awesome contribution!

**Pull request merged**